index.html

```diff
@@ -0,0 +1,471 @@
+            <a href="">
+              <img src="./images/dot.jpg" alt="dot2">
+            </a>
+          </li>
+
+          <li class="year">
+            <a href="">
+              <img src="./images/2015.jpg" alt="year">
+            </a>
+          </li>
+
+        </ul>
+
+        <div class="content_paragraph">
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+
+        <div class="tech_list">
+          <ul class="list">
+            <li>
+              <a href="">
+                <img src="./images/htmltag.jpg" alt="html">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/csstag.jpg" alt="css">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/jstag.jpg" alt="js">
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <a href="">
+          <img class="button" src="./images/SeeProject.jpg" alt="">
+        </a>
+      </div>
+    </section>
+
+    <section class="container5">
+      <div class="Pics">
+        <img src="./images/aboutmyselftitle.jpg" alt="title">
+      </div>
+      <div class="text">
+        <img src="./images/second_text.jpg" alt="text">
+      </div>
+
+      <p class="third-para">LET’S CONNECT</p>
+
+      <ul class="social_links2">
+
+
+        <li><a href=""><img src="./images/Vector-0.jpg" class="logo-icons2" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Linkedin-icon.jpg" class="logo-icons2" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Vector-4.jpg" class="logo-icons2" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Vector-2.jpg" class="logo-icons2" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Vector-3.jpg" class="logo-icons2" alt="union" /></a></li>
+      </ul>
+
+      <div>
+        <a href="#">
+          <img class="get_my_resume" src="./images/Getmyresume.jpg" alt="get_my_resume">
+        </a>
+      </div>
+
+      <div>
+        <img class="languages_logo" src="./images/Languages.jpg" alt="languages">
+      </div>
+
+      <div>
+
+        <ul class="languages">
+          <li>
+            <a href="#">
+              <img src="./images/jsframe.jpg" alt="js">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/htmlframe.jpg" alt="html">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/cssframe.jpg" alt="css">
+            </a>
+          </li>
+        </ul>
+
+        <ul class="frame">
+          <li>
+            <a href="#">
+              <img class="frame" src="./images/Frameworks.jpg" alt="frame">
+            </a>
+          </li>
+        </ul>
+
+        <hr>
+        <ul>
+          <li>
```

Comment:
- [ ] Please ensure that the `languages` section design is properly implemented to align to the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/165339114-fcf97f8e-a4c3-49b9-a7f1-4c7fc034428e.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165339283-2333b6cc-8989-46b1-b0ab-addfc3cf5b71.png)